core/sdk-core/src/main/java/software/amazon/awssdk/core/retry/backoff/SdkDefaultBackoffStrategy.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.core.retry.backoff;
+
+import java.time.Duration;
+import software.amazon.awssdk.core.internal.retry.SdkDefaultRetrySetting;
+import software.amazon.awssdk.core.retry.RetryPolicyContext;
+import software.amazon.awssdk.core.retry.RetryUtils;
+
+public class SdkDefaultBackoffStrategy implements BackoffStrategy {
+
+    private final FullJitterBackoffStrategy defaultBackoffStrategy = FullJitterBackoffStrategy.builder()
+                                        .baseDelay(SdkDefaultRetrySetting.BASE_DELAY)
+                                        .maxBackoffTime(SdkDefaultRetrySetting.MAX_BACKOFF)
+                                        .build();
+
+    private final EqualJitterBackoffStrategy throttledBackoffStrategy = EqualJitterBackoffStrategy.builder()
+                                        .baseDelay(SdkDefaultRetrySetting.THROTTLED_BASE_DELAY)
+                                        .maxBackoffTime(SdkDefaultRetrySetting.MAX_BACKOFF)
+                                        .build();
+
+    @Override
+    public Duration computeDelayBeforeNextRetry(RetryPolicyContext context) {
+        if (RetryUtils.isThrottlingException(context.exception())) {
```

Comment:
What do you think adding `throttlingBackoffStrategy` API on `RetryPolicy` or in this class so that all clients can default get this behavior with custom backoffstrategy?

Otherwise, we'd have to copy the same code in `DynamodbRetryPolicy`

https://github.com/aws/aws-sdk-java-v2/blob/a0ae6d620efb25a6fec77e5879e443baccac15bf/services/dynamodb/src/main/java/software/amazon/awssdk/services/dynamodb/DynamoDbRetryPolicy.java#L45-L50

Reply:
Yeah, I had considered this as well. Let me write it up and see how it feels.